frontend/src/views/Game.tsx

```diff
@@ -147,8 +148,28 @@ function GamePage() {
     setTimeUp(newGame.gameTimer.timeUp);
   };
 
+  const createCodeLanguageArray = () => {
+    while (currentLanguage.length < problems.length) {
+      currentLanguage.push(Language.Python);
+    }
+
+    while (currentCode.length < problems.length) {
+      currentCode.push('');
+    }
+  };
+
+  createCodeLanguageArray();
```

Comment:
I'm not sure if this actually works - if I change the language to Java for any problem, then the rest will also be Java. I think instead you could put this into a useEffect hook (with a comment explaining that this initializes these values) for whenever problems changes (that way this also isn't repeatedly called on every re-render

Reply:
So the codeLangeArray is just the initial array. Afterwards, the language select will only change one of the elements in the array. This can be checked in 147. Although, I'm not sure what to do about the re-render problem. I'm afraid that if I put it in a useEffect hook that it'll overwrite itself accidentally, as the hook would be called every time the languageArray or codeArray is changed.